src/bin/smbios-battery-ctl

```diff
@@ -77,6 +80,70 @@ def print_express_charge_status(n):
     elif n == 0xFF: print('\t Battery is present, and it does not support express charge')
     else : print('\t Unknown')
 
+class ChargingMode(Enum):
+    PRIMARILY_AC_USE = 0x0341
+    ADAPTIVE_CHARGE = 0x0342
+    CUSTOM_CHARGE = 0x0343   # TODO: or 0x0348? both work on my system
```

Comment:
0x343

Reply:
Out of curiosity, why are there two tokens for that setting?

Reply:
I don't know the history of why, but one is marked deprecated.

Reply:
Where is it marked deprecated? I'd like reference the source in the comment.

Reply:
It's not in a public document, you can feel free to add a (deprecated) to the token in https://github.com/dell/libsmbios/blob/master/doc/token_list.csv as part of your commit though.